src/flows/pipes/Notification/view.tsx

```diff
@@ -481,7 +482,9 @@ const Notification: FC<PipeProp> = ({Context}) => {
         </FlexBox>
         <Panel.Footer justifyContent={JustifyContent.FlexEnd}>
           <FlexBox margin={ComponentSize.Medium}>
-            <ExportTask />
+            <ErrorBoundary>
+              <ExportTask />
+            </ErrorBoundary>
```

Comment:
ExportTask has many uses of the lsp helper methods, which are not all error handled. Therefore, adding an ErrorBoundary wrapper.